src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java

```diff
@@ -0,0 +1,19 @@
+package api.java.openiss.ws.soap.service;
+import javax.jws.WebMethod;
+import javax.jws.WebParam;
+import javax.jws.WebService;
+import javax.jws.soap.SOAPBinding;
+import javax.xml.ws.BindingType;
+
+@WebService
+@BindingType(value = "http://java.sun.com/xml/ns/jaxws/2003/05/soap/bindings/HTTP/")
+@SOAPBinding(style = SOAPBinding.Style.RPC)
+public interface OpenISSSOAPService {
+
+    @WebMethod
+    byte[] getFrame(@WebParam(name = "type") String type);
+
+    @WebMethod
+    String getFileName(@WebParam(name = "type") String type);
```

Comment:
(I) `getFileName()` WebMethod was not in the spec; why do you need this service call? At the same time, the required `mixFrame()` service method is not even defined in the interface and implemented as a stub. I understand you may not have had the time to deal with it, but its declaration should still be in the interface.